Unity3D/Assets/RosSharp/Scripts/JoyPublisher.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
```

Comment:
Please change this file to a non-Oculus version using e.g.
* https://docs.unity3d.com/ScriptReference/Input.GetAxis.html
* https://docs.unity3d.com/ScriptReference/Input.GetKey.html
or remove it form the pull request